be/src/formats/csv/converter.h

```diff
@@ -74,6 +74,8 @@ class Converter {
 };
 
 std::unique_ptr<Converter> get_converter(const TypeDescriptor& type_desc, bool nullable);
+std::unique_ptr<Converter> get_converter(const TypeDescriptor& type_desc, bool nullable, char collection_delimiter,
```

Comment:
I think one `get_converter` is enough, you can give `collection_delimiter` a default value.

Reply:
+1

> I think one `get_converter` is enough, you can give `collection_delimiter` a default value.

Reply:
I had to rewrite a new version. PTAL.